spec/main_rspec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../lib/manipulation'
+
+
+describe Parser do
+    it 'file exists' do
+        expect(File.exist?('games.log')).to be(true)
+    end
+
+    it 'print first line' do
+        data = File.open('games.log')
+        expect(data.readline.rstrip).to match("  0:00 ------------------------------------------------------------")
+        data.close
```

Comment:
Aqui basicamente não tá sendo testado o comportamento da sua classe `Parser`, visto que você tá só replicando o comportamento interno da mesma. A ideia do teste é que você consiga testar o código que você fez, invocando os métodos dele e não duplicando o código dentro do teste. Ficaria mais ou menos dessa forma: 

```suggestion
        data = Parser.new('games.log')
        expect(data.openfile).to match("  0:00 ------------------------------------------------------------")
```